scripts/build.py

```diff
@@ -141,6 +165,8 @@ def get_cache_slug():
     CACHE_SLUG = get_cache_slug()
     BUILD_DIR = os.path.join('build', CACHE_SLUG)
 
+    # ensure_directory_exists trims file paths passed to it. Hence, directory
+    # paths require a trailing slash.
```

Comment:
I'm not sure what this comment is doing here; it doesn't seem to be describing any code?